build.gradle.kts

```diff
@@ -1,6 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id("com.android.application") version "8.3.0" apply false
+    id("com.android.application") version "8.3.1" apply false
```

Comment:
Why is a gradle file committed?

Reply:
it really shouldn't be, this is something that we should probably add to the gitignore file later